### ISSUE TITLE
1.19 regression with relative paths

### ISSUE THREAD

**Issue by spagh-eddie** [lnoor/sphinx-jsonschema#76]
## issue

our code had RST files like this which worked in version 1.18
```rst
.. jsonschema:: ../../../path/to/schema.json
```
but now they throw errors like this
```console
reading sources... [100%] recipe/protocol-1                                                                                                                                                
/Users/edoyle/Desktop/spagh-eddie/icsw/docs/source/recipe/base.rst:8: ERROR: Traceback (most recent call last):
  File "/Users/edoyle/.pyenv/versions/3.10.1/lib/python3.10/importlib/__init__.py", line 121, in import_module
    raise TypeError(msg.format(name))
TypeError: the 'package' argument is required to perform a relative import for '../../../icsw/recipe/schemas/base'
```

I have found the issue to be due to e6177b06363808eed610a8e5f46cd243f65e35be, which changed how `filename` is handled. My relative paths were getting skipped over since the `exists(filename)` would return `False` without the `os.path.join(document_source, filename)`!

**Comment by kloczek** [comment]
I see that yestreday released new version is tagged as `v.19.0`.
Q: is it not a typo and it should be `v.0.19.0`? 🤔

**Comment by kloczek** [comment]
And on pypi it is published as `1.19.0` https://pypi.org/project/sphinx-jsonschema/ ..
